docs/edge.rst

```diff
@@ -0,0 +1,253 @@
+.. _edge:
+
+============
+CrateDB Edge
+============
+
+Crate.io is pleased to `announce CrateDB Edge`_, the hybrid cloud database
+solution integrating local control over regions and data storage with the
+managed software stack from CrateDB Cloud.
+
+The process of getting CrateDB Edge running is less complicated than it may
+seem, thanks to the support for Edge deployment built into the CrateDB Cloud
+Console, our own web UI. Even so, there are some steps involved, and some
+requirements have to be met in order for it to work. This tutorial therefore
+serves as an end-to-end walkthrough of the process and of these prerequisites.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _edge-prereqs:
+
+Prerequisites
+=============
+
+Certain hardware and software specifications must be met in order to make use
+of CrateDB Edge. The most important of these is that you must provide a working
+Kubernetes cluster, one that meets the following requirements:
+
+.. rst-class:: open
+
+* It must contain at least three nodes (for high availability);
```

Comment:
Is this requirement enforced? What if e.g. we only provide a K8s cluster with one node?

Reply:
@SStorm or @WalBeh will probably be able to confirm this but as far as I know there are no plans for 1 single scale unit and I _think_ the CrateDB Kubernetes configuration enforces each pod to run on a different K8s node. If that is true then it is an "enforced" requirement indeed.

On the other hand, no production environment should run in less than 3 nodes and we should make this clear. It is a database after all and high availability and data resiliency are a must.